### PR TITLE
feat: add error 404 page

### PR DESCRIPTION
**Pull request by him-raw99**
## Related Issue of adding a custom error404 page

Closes: #698 

#### Screenshots 

image

kindly comment if you find something wrong with this branch

### CODE REVIEW

**Comment by him-raw99** [comment]
> We require all PRs to follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/). More details 👇🏼
> 
> ```
>  No release type found in pull request title "Add error 404 page". Add a prefix to indicate what kind of release this pull request corresponds to. For reference, see https://www.conventionalcommits.org/
> 
> Available types:
>  - feat: A new feature
>  - fix: A bug fix
>  - docs: Documentation only changes
>  - style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
>  - refactor: A code change that neither fixes a bug nor adds a feature
>  - perf: A code change that improves performance
>  - test: Adding missing tests or correcting existing tests
>  - build: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
>  - ci: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
>  - chore: Other changes that don't modify src or test files
>  - revert: Reverts a previous commit
> ```

I am sorry, I have updated it now. thanks for you patience.

File: src/assets/pictures/error404.svg
Comment (IAmTamal) [inline src/assets/pictures/error404.svg:1]: The SVG file is too big on screen, please reduce it - so that it's centered and we all enjoy a Minimal vibe !

**Review by IAmTamal — CHANGES_REQUESTED** [review]
The rest looks good to me.

File: src/assets/pictures/error404.svg
Comment (him-raw99) [reply src/assets/pictures/error404.svg:1]: on it.

**Comment by him-raw99** [comment]
>I have reduced the size of SVG file to about 50% of screen width and its centered.

>along with that I have changed the button text to just "HOMEPAGE" to improve the overall dynamics of the web-page.

kindly let me know if it needs some more work.

**Review by IAmTamal — APPROVED** [review]
**Amazing work,**
This looks good to me, will be merged soon 🎉.

**Pull request merged**